sources/Launcher/DYNMarginCalculationLauncher.cpp

```diff
@@ -804,7 +804,8 @@ MarginCalculationLauncher::launchLoadIncrease(const boost::shared_ptr<LoadIncrea
   dumpFile << "loadIncreaseFinalState-" << variation << ".dmp";
   params.dumpFinalStateFile_ = createAbsolutePath(dumpFile.str(), workingDirectory_);
 
-  result.setScenarioId(computeLoadIncreaseScenarioId(variation));
+  result.setScenarioId("loadIncrease");
```

Comment:
Why changing the name?

Reply:
then we should change the way to generate the file name as this modification might change the others log

Reply:
It doesn't seem logical to me. The actual scenario id of the load increase is "loadIncrease" in fic_MULTIPLE.xml (it's the same for "DisconnectLine" and "DisconnectGroup" which don't include the variation in their name).

The names loadIncrease-67, loadIncrease-68,... which include the variation in them are used for the load increase simulation part in multiprocessing.

Reply:
I justtook a loog and this naming is critical in multi processing as it is the way used to retrieve a loadincrease ran by another process

Reply:
We can keep this way to retrieve load increase results in multiprocessing but I don't see the point to keep the same convention to name the scenarioId. And what I made works.

Reply:
I though this modification was supposed to be reverted?